javaslang/src/main/java/javaslang/collection/Array.java

```diff
@@ -736,7 +736,7 @@ public T get(int index) {
                     list.add(u);
                 }
             }
-            return wrap(toArray(list));
+            return wrap(list.toArray());
```

Comment:
I think it would be more rezistent to change via the general (warmed up) method, but I don't mind leaving it this way either.